src/Pages/Landing.js

```diff
@@ -25,7 +25,8 @@ function Landing() {
         unapproved.push(js);
       });
     } catch (error) {
-      console.log(error);
+      // eslint-disable-next-line no-alert
```

Comment:
can we either configure an es lint to ignore alerts or alternatively just take the warning? or does it throw a console error